community/asking-questions.md

```diff
@@ -0,0 +1,12 @@
+# Asking Questions on the Forum
+
+Author(s): code3z (@code3 on the forum)
+
+The [modding forum](https://forum.vivaldi.net/category/52/modifications) is also a great place to ask questions about modding. Before asking, search the forum and the internet to make sure there is not already a good answer.
+
+When asking a question, keep a few simple things in mind:
+ - Use a topic title that is descriptive
+ - Be polite and don't use ALL CAPS
+ - If you are asking questions or trying to debug a certain piece of code, share that code
```

Comment:
> If you are asking questions or trying to debug a certain piece of code

‘if you are asking about a certain piece of code’?

Reply:
Sure.